### ISSUE TITLE
Finding the right "entry"

### ISSUE THREAD

**Issue by wojosc** [tueit/iCal-integration#2]
Hey @PawanMeh I just had a thought which might help alot:

iCal Subscription could work very similar to https://docs.erpnext.com/docs/user/manual/en/setting-up/email/auto-email-reports

Are you using that? If you need an introduction, let me know.

Best
Wolfram

**Comment by PawanMeh** [comment]
@wojosc I think you may just want to include user as a link field on the iCal subscription page and then get the events associated with the user? I am not sure how auto email or something similar could be used here.

**Comment by wojosc** [comment]
@PawanMeh yeah, a user field will probably do the job. This could be used just the way as it is used in auto e-mail "Based on User Permission".

So when I have a User set in iCal Subscription he will just see events which he would be able to see logged into the system and based on the doctype selected.
